src/main/include/HWConfig.hpp

```diff
@@ -2,4 +2,36 @@
 
 #pragma once
 
-namespace frc3512::HWConfig {}  // namespace frc3512::HWConfig
+namespace frc3512 {
+
+namespace HWConfig {
+
+/// ID for the intake motor
+constexpr int kIntakeMotorID = 0;
+
+/// Channal for the Fourbar
+constexpr int kFourbarChannel = 1;
+
+/// ID for the conveyor motor
+constexpr int kConveyorMotorID = 2;
+
+/// ID for the Left Telescoping Motor
+constexpr int kLeftTeleMotorID = 3;
+
+/// ID for the Right Telescoping Motor
+constexpr int kRightTeleMotorID = 4;
+
+/// ID for the Left climber Solenoid
+constexpr int kLeftTeleSolenoid = 5;
+
+/// ID for the Right climber Solenoid
+constexpr int kRightTeleSolenoid = 6;
+
+/// Temp ID for appendageStick1
+constexpr int appendageStick1PortID = 7;
+
+/// Temp ID of joyStick1
+constexpr int joyStick1PortID = 8;
```

Comment:
Same argument with the joystick (I'm aware its temporary. However, it helps in us designing the button layout.)